website/tosti/templates/tosti/footer_buttons.html

```diff
@@ -0,0 +1,17 @@
+<footer class="page-footer navbar navbar-expand-md fixed-bottom">
+    <div class="container text-center">
+        <div class="row flex-grow-1">
+            {% for button in buttons %}
+                <div class="col">
+                    <div class="btn-ml btn-on m-auto">
+                        {% if button.icon %}
+                            <i class="fas fa-{{ button.icon }}"></i>
```

Comment:
Okay that comment should have been here actually

Reply:
Nice code though

Reply:
This file should have been deleted :laughingstocks